structure/type.go

```diff
@@ -48,13 +48,6 @@ func (t *TxStructure) encodeStringDataKey(key []byte) kv.Key {
 	return codec.EncodeUint(ek, uint64(StringData))
 }
 
-func (t *TxStructure) encodeHashMetaKey(key []byte) kv.Key {
```

Comment:
It's used by TestHash(), but the test is not enabled now.

Reply:
OK, I will revert this one.